credentials/apps/edx_credentials_extensions/edly_credentials_app/admin.py

```diff
@@ -0,0 +1 @@
+from django.contrib import admin
```

Comment:
we don't necessarily need to create an admin file if we are not going to register a model

Reply:
Kept it there if in future we need to use this.